gapis/config/config.go

```diff
@@ -16,25 +16,53 @@
 package config
 
 const (
-	DebugReplay                = false
-	DebugReplayBuilder         = false
+	// DebugReplay activates various debug logs related to replay.
+	DebugReplay = false
+
+	// DebugReplayBuilder activates various debug logs and checks related to the
+	// creation of a replay payload.
+	DebugReplayBuilder = false
+
+	// DisableDeadCodeElimination prevents the early computation of the
+	// dependency graph.
 	DisableDeadCodeElimination = false
-	DeadSubCmdElimination      = false
-	DebugDeadCodeElimination   = false
-	DebugDependencyGraph       = false
-	DumpReplayProfile          = false
-	DumpValidationTrace        = true
-	AllInitialCommandsLive     = false
-	LogExtrasInTransforms      = false // Logs all commands' extras together with transforms
-	LogMemoryInExtras          = false // Logs all commands' read/write memory observation together with extras
-	// Logs all mappings at the end of the replay from original trace
-	// handles to replay client handles (if handles are reused in the trace
-	// it will only print the last mapping).  Only works for Vulkan.
-	LogMappingsToFile        = false
-	LogTransformsToFile      = false
-	LogTransformsToCapture   = false
-	LogInitialCmdsIssues     = false
-	LogInitialCmdsToCapture  = false
-	SeparateMutateStates     = false
-	CheckRebuiltStateMatches = false
+
+	// DeadSubCmdElimination prevents the elimination of subcommands in DCE.
```

Comment:
nit: Using the DCE in the comment caused me to stumble a bit while reading and trying to figure out what DCE means. From the other options it becomes obvious that it is "dead code elimination". 

Options:
(1) Always spell out "dead code elimination" instead of DCE. This is somewhat my preference, as it keeps the flow while reading and I don't see much downside of the slightly more verbose comment.
(2) Introduce DCE as an abbreviation above, e.g. "prevents the early computation of the dependency graph, which is used to perform dead code elimination (DCE)."

Option (1) would apply to all uses of DCE here. I also prefer option 1 as the reader may just arrive here looking at a specific option, and doesn't read from top to bottom, i.e. even if we introduce the abbreviation, they may miss it.

Reply:
Good point, I've spelled it out entirely here. There's a bit of a balance between avoiding acronyms and using ones that are common enough, but I've very much on the side of avoiding them for the sake of clarity and accessibility to our code.

Note that you may see DCE in other parts of the code, I didn't want to replace all occurrences as that's probably a wide change.